R/project.R

```diff
@@ -55,11 +74,85 @@ setMethod("project", "MAXENT.SDM", function(obj, Env, ...) {
   # Rescaling projection
   proj = reclassify(proj, c(-Inf, 0, 0))
   if(!all(obj@data$Presence %in% c(0,1))) # MEMs should not be rescaled
-    if(proj@data@max > 0) proj = proj / proj@data@max
+    if(proj@data@max) proj = proj / proj@data@max
   names(proj) = "Projection"
   obj@projection = proj
   if(all(obj@data$Presence %in% c(0,1))) # MEMs can't produce binary
     obj@binary <- reclassify(proj, c(-Inf,obj@evaluation$threshold,0,
                                      obj@evaluation$threshold,Inf,1))
   return(obj)
 })
+
+#' @rdname project
+#' @export
+setMethod("project", "Ensemble.SDM", function(obj, Env, ...) {
```

Comment:
I'll add the unit tests for project to the existing `modelling`, `ensemble_modelling` and `stack_modelling` tests if that's ok (otherwise we would have to build them again just to apply project)?

Reply:
I see, you want to save testing time. But doing so if something breaks we will be less able to see where the issue come from (e.g. if it's the `ensemble_modelling` our the `project` that had a bug). As the test are currently really short do it this way. But maybe in a near future we could do a unit test dedicated to project (for SDM, ESDM, SSDM with current or new environments).

Reply:
No, you're right, I'll write a new test. There's no sense in postponing.